data/coalition/coalition missions.txt

```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+			choice
+				`	"Alright, let's go!"`
+					goto questions
+				`	"I don't want to do this anymore."`
+					goto scared
+			label scared
+			`	"To answer their questions, if you really don't want, I can't stop you, I suppose." Aelbris says with a sigh.`
+					decline
+			label questions
+			`	As soon as you walk up to the stage, several scientists in the crowd stand up out of their chairs. An Arach comes up to you on the stage, and explains that this is their way of asking questions. She selects a Saryd from the crowd.`
+			branch "hai"
+				has "First Contact: Hai: offered"
+			`	"Aliens besides the Quarg, are humans in contact with?"`
+			choice
+				`	"No, the only aliens humans know about are the Quarg."`
+					goto "sad"
+			label "hai"
+			`	"Aliens besides the Quarg, are humans in contact with?"`
+			choice
+				`	"Yes, humans are in contact with the Hai. They live to the north of human space, and we can visit them through a wormhole."`
+					goto "know"
+				`	"A few humans know about the Hai, but there's no official acknowledgement of their existence. They live to the north of human space, and we can visit them through a wormhole."`
+					goto "know"
+				`	"No, the only aliens humans know about are the Quarg."`
+					goto "sad"
+			label "sad"
+			`	At this several in the audience sound disappointed.`
+				goto "nextquestion"
+			label "know"
+			`	At this several in the audience sound delighted. "Cooperation between different species, it is important to foster," says the Saryd who asked the question. An Arach asks you to point out where the Hai are on a galactic map, and you do so.`
+			label "nextquestion"
+			branch "war"
+				has "FWC End: completed"
+			`	The Arach selects another, this time a Kimek. "All of human space, united politically is it?"`
+			choice
+				`	"No, human space is split between the Republic in the north and the Free Worlds in the south."`
+				`	"No, human space is split between the Republic in the west, the Free Worlds in the south, and the Syndicate in the east. The Syndicate is technically part of the Republic, but they're very autonomous."`
+			`	There are murmurs in the crowd, and several begin to write down notes on various devices."`
+				goto "questionthree"
+			label "war"
+			`	The Arach selects another, this time a Kimek. "All of human space, united politically is it?"`
+			choice
+				`	"Yes, all of human space is united under the Republic."`
+				`	"Yes, but there are internal subdivisions: the Free Worlds in the south, the Republic in the north, and the Syndicate in the east."`
```

Comment:
This is _technically_ untrue if the player chooses FW Checkmate because that results in them being autonomous from the Republic.

Reply:
To me, 'internal subdivisions' mean 'These divisions have their own system of government and politics'. To be honest with you though, I've never actually played Checkmate.

Reply:
I think that's fair- feel free to resolve.